test/prompt.test.ts

```diff
@@ -0,0 +1,33 @@
+import cli from '../src'
+
+import {expect, fancy} from './fancy'
+
+describe('prompt', () => {
+  after(function () {
+    (global as any).asyncDump()
```

Comment:
what is this?

Reply:
Removed. It was some code to help me debug the hanging promise.